tests/unit/full_flow/test_full_flow.py

```diff
@@ -30,6 +32,26 @@
         "DEBUG": "false",
     })
 @pytest.mark.parametrize("testcase", [
+    ({
+        "lambda_event_file": THIS_DIR / "../../../examples/app_runner/kinesis_request_app_runner_application.json",
```

Comment:
I think you went into wrong place with the test, please check out https://github.com/dynatrace-oss/dynatrace-aws-log-forwarder/blob/master/tests/unit/logs/test_transformation_full.py 
This has metadata attributes validation

New config I think does not need this ful_flow test

Reply:
oh! damn 😂 Now I see... I missed tests in the `logs` subfolder
OK, I revert changes in the full flow test and extend the transformation test.